doc/rippled-example.cfg

```diff
@@ -987,23 +972,6 @@ pool.ntp.org
 [ips]
 r.ripple.com 51235
 
-# Public keys of the validators that this rippled instance trusts.  The latest
```

Comment:
@wilsonianb:

- There's no logging because `j_` is default initialized (and thus, outputs nothing). You can throw if you want, but make sure the throw message explains the issue.
- I'm OK with allowing the current path to be searched for the specified file, but only if the specified path is _not_ absolute (see: http://www.boost.org/doc/libs/1_60_0/libs/filesystem/doc/reference.html#path-is_absolute)
- I'm OK with having the default config include a `[validators_file]` entry that points to `validators.txt` (since that's the the default behavior).

Again, I'll point out that these changes make `rippled` harder to configure, but this is a server so I'm not too concerned.